client/src/Components/Pages/UpdateAlert.js

```diff
@@ -1,14 +1,145 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import axios from "axios";
+import Select from "react-select";
+import { updateAlert } from "../../actions/alerts";
+import { useDispatch, useSelector } from "react-redux";
+import webHost from '../../config/web';
 
+import api from "../../config/web";
 
-const DUMMY_ALERTS = [
-    {
-        
+const UpdateAlert = () => {
+  const [alertData, setAlertData] = useState({});
+  const [symbols, setSymbols] = useState([]);
+
+  const dispatch = useDispatch();
+  const {id} = useParams();
+
+  const history = useHistory();
+
+  useEffect(() => {
+    const fetchSymbolsList = () => {
+      axios
+        .get("https://api.binance.com/api/v1/exchangeInfo")
```

Comment:
I think the binance API url should come from somewhere else so you can do this 

`axios.get(`${BINANCE_URL}/exchangeInfo`)

Reply:
That allows you to re-use the URL in mutliple spots. This in important, because let's say you decide to use v2 of the binance API at some point, you only have to make that change in one place

Reply:
Done